src/components/Preferences/Availability.tsx

```diff
@@ -53,22 +50,27 @@ const Availability = () => {
                     </Checkbox>
                     <HStack spacing={5}>
                         <Text minW="180px"> If yes, enter the duration:</Text>
-                        <Select
-                            isDisabled={isSabbaticalDisabled}
-                        >
+                        <Select isDisabled={isSabbaticalDisabled}>
                             <option value="option1">half leave</option>
                             <option value="option2">full leave</option>
                         </Select>
                         <Text>from</Text>
-                        <Select
-                            isDisabled={isSabbaticalDisabled}
-                        >
+                        <Select isDisabled={isSabbaticalDisabled}>
                             <option value="option1">January</option>
                             <option value="option2">May</option>
                             <option value="option3">September</option>
                         </Select>
                     </HStack>
                 </Stack>
+                <HStack spacing={5}>
+                    <Text>How many days a week would you like to teach?</Text>
+                    <NumInput id="days" />
+                    <Checkbox>Mon</Checkbox>
```

Comment:
Do these need labels for data tracking?

Reply:
Maybe value or [onChange](https://chakra-ui.com/docs/components/form/checkbox#indeterminate) props... can probably leave until we start integrating the form

Reply:
https://chakra-ui.com/guides/integrations/with-formik#:~:text=about%20using%20the-,Formik%20component,-instead%20of%20the

https://formik.org/docs/api/field